templates/workspace_services/azureml/user_resources/aml_compute/porter.yaml

```diff
@@ -55,30 +56,32 @@ mixins:
 
 
 install:
-   - terraform:
-      description: "Deploy service"
-      vars:
-        workspace_id: "{{ bundle.parameters.workspace_id }}"
-        tre_id: "{{ bundle.parameters.tre_id }}"
-        tre_resource_id: "{{ bundle.parameters.id }}"
-        parent_service_id: "{{ bundle.parameters.parent_service_id }}"
-        vm_size_sku: "{{ bundle.parameters.vm_size }}"
-        auth_tenant_id: "{{ bundle.parameters.auth_tenant_id }}"
-        user_object_id: "{{ bundle.parameters.user_object_id }}"
-        arm_tenant_id: "{{ bundle.credentials.azure_tenant_id }}"
-        arm_client_id: "{{ bundle.credentials.azure_client_id }}"
-        arm_client_secret:  "{{ bundle.credentials.azure_client_secret }}"
-        arm_use_msi: "{{ bundle.parameters.arm_use_msi }}"
-      backendConfig:
-        resource_group_name: "{{ bundle.parameters.tfstate_resource_group_name }}"
-        storage_account_name: "{{ bundle.parameters.tfstate_storage_account_name }}"
-        container_name: "{{ bundle.parameters.tfstate_container_name }}"
-        key: "tre-user-resource-aml-compute-instance-{{ bundle.parameters.id }}"
+  - terraform:
```

Comment:
@ross-p-smith git another issue here, should be an indent after `- terraform` as per:
<img width="182" alt="image" src="https://user-images.githubusercontent.com/17089773/151942521-cab5d605-281e-4dde-add1-2521f20de3dd.png">

https://porter.sh/mixins/terraform/

I can get this fixed, but would have thought this would have been picked up when registering bundles in CI/CD. Is this bundle being registered?